### PR TITLE
feat : added archive notes functionality

### PR DESCRIPTION
**Pull request by GeetaPogul**

### CODE REVIEW

File: src/context/DelNote-context.js [hunk 1]
```diff
@@ -0,0 +1,34 @@
...
+    >
+      {children}
+    </DelNoteContext.Provider>
+  );
+};
+export function useDelNote() {
+  return useContext(DelNoteContext);
+}
+
+//  {useDelNote, DelNoteProvider}
```
Comment (nakulsharma15) [inline src/context/DelNote-context.js:34]: Remove commented code.

File: src/css/login.css [hunk 2]
```diff
@@ -0,0 +1,59 @@
+.page-container{
+    background-color: white;
```
Comment (nakulsharma15) [inline src/css/login.css:2]: You can use variables for colors

File: src/css/login.css [hunk 3]
```diff
@@ -0,0 +1,59 @@
...
+    width: max-content;
+    line-height: 1.7rem;
+    margin: auto;
+    color : var(--text-color);
+    border-radius: .2rem;
+    /* box-shadow: var(--shadow-color); */
```
Comment (nakulsharma15) [inline src/css/login.css:8]: Remove commented code.

File: src/pages/Login.js [hunk 4]
```diff
@@ -0,0 +1,67 @@
...
+              Email Address <span className="star"> *</span>
+            </label>
+            <br />
+            <input
+              className="input-box"
+              type="email"
+              placeholder="johndoe@example.com"
+            />
+            <br />
+            <br />
```
Comment (nakulsharma15) [inline src/pages/Login.js:23]: Avoid using br  tag

File: src/pages/DemoNote.js [hunk 5]
```diff
@@ -0,0 +1,53 @@
...
+  console.log(dateStr);
+
+  return (
+    <>
+      <div className="note-container">
+        <div className="new-note">
+          <div className="text">
+            <div className="btns-end">
+              <button className="btn btn-outline-warning icon-btns ">
+                {" "}
```
Comment (nakulsharma15) [inline src/pages/DemoNote.js:26]: Remove empty strings as it's not being used.

File: src/pages/Home.js [hunk 6]
```diff
@@ -0,0 +1,27 @@
+import React from "react";
+import { Link } from "react-router-dom";
+import "../css/home.css";
+const Home = () => {
+  return (
+    <>
+      {/* <h1> Home Page</h1> */}
```
Comment (nakulsharma15) [inline src/pages/Home.js:7]: Remove commented code.

File: src/pages/Nav.js [hunk 7]
```diff
@@ -0,0 +1,67 @@
...
+
+import { Link } from "react-router-dom";
+// import Home from "./Home";
+// import Docs from "./Docs";
+// import Label from "./Label";
+// import Login from "./Login";
+// import Notes from "./Notes";
+// import Archive from "./Archive";
+// import Trash from "./Trash";
+// import Signup from "./Signup";
```
Comment (nakulsharma15) [inline src/pages/Nav.js:12]: Remove commented code since it's not needed.

File: src/pages/Nav.js [hunk 8]
```diff
@@ -0,0 +1,67 @@
...
+      {/* <Routes>
+        <Route path="/home" element={<Home />} />
+        <Route path="/docs" element={<Docs />} />
+        <Route path="/login" element={<Login />} />
+        <Route path="/label" element={<Label />} />
+        <Route path="/notes" element={<Notes />} />
+        <Route path="/trash" element={<Trash />} />
+        <Route path="/archive" element={<Archive />} />
+        <Route path="/signup" element={<Signup />} />
+      </Routes> */}
```
Comment (nakulsharma15) [inline src/pages/Nav.js:62]: Same as above.

File: src/pages/Signup.js [hunk 9]
```diff
@@ -0,0 +1,62 @@
...
+  return (
+    <>
+      SIGN UP Page
+      <div className="page-container"> 
+      <form>
+        <label className="input-label">
+          Name
+          <span className="star"> *</span>
+        </label>
+        <br />
```
Comment (nakulsharma15) [inline src/pages/Signup.js:14]: Avoid using < br > tags

File: src/pages/Notes.js [hunk 10]
```diff
@@ -0,0 +1,63 @@
...
+
+  return (
+    <>
+      <div className="wrapper">
+        <div className="slider">
+          <SubLink />
+        </div>
+        <div className="slider-content">
+          <AddNotes passNote={addNewNote} className="create-note-div" />
+          <br />
```
Comment (nakulsharma15) [inline src/pages/Notes.js:39]: Avoid using br tags

**Review by nakulsharma15 — COMMENTED** [review]
Suggested some changes. Everything else seems good. I Liked how you named the variables and functions

File: src/pages/AddNotes.js [hunk 11]
```diff
@@ -0,0 +1,76 @@
...
+    });
+  };
+
+  const expandNoteBox = () => {
+    setExpand(true);
+  };
+
+  const backToShrink = () => {
+    setExpand(false);
+  };
```
Comment (gopalbbharadva) [inline src/pages/AddNotes.js:38]: Avoid curly braces, it can be one line functions

File: src/pages/Nav.js [hunk 12]
```diff
@@ -1,18 +1,19 @@
...
+import { Route, Routes, Link } from "react-router-dom";
 import Home from "./Home";
 import Docs from "./Docs";
 import Label from "./Label";
 import Login from "./Login";
 import Notes from "./Notes";
-import Remainder from "./Remainder";
-import Trash from "./Trash"
+import Archive from "./Archive";
+import Trash from "./Trash";
```
Comment (gopalbbharadva) [inline src/pages/Nav.js]: Suggestion: You can import all these in one file , export from that single file and then import it here.

**Review by gopalbbharadva — APPROVED** [review]

**Review by Jay-Gadhiya — APPROVED** [review]

**Pull request merged**